core/src/main/java/org/infinispan/persistence/PersistenceUtil.java

```diff
@@ -129,100 +129,101 @@ public static InternalMetadata internalMetadata(InternalCacheValue icv) {
             isLoaded);
    }
 
-   public static <K, V> InternalCacheEntry<K,V> loadAndStoreInDataContainer(DataContainer<K, V> dataContainer, int segment,
```

Comment:
The package is `@public`, so I'm not sure about these API changes

Reply:
I know the diff is really ugly, but the public methods are still there. If not then that is an issue.